### ISSUE TITLE
 Error updating pod: The platform of the target `MY APP NAME` (iOS 9.0) is not compatible with `WillowTreeReachability (2.0.2)`, which does not support `ios`.

### ISSUE THREAD

**Issue by fabioiegri** [willowtreeapps/WillowTreeReachability#10]
Hello All and Thank You for Your Work.
I have a problem updating the pod.
Launching _pod update_ command in my iOS app, I receive the following error:
```
Analyzing dependencies
[!] The platform of the target `MY APP NAME` (iOS 9.0) is not compatible with `WillowTreeReachability (2.0.2)`, which does not support `ios`.
```

In my Podfile I tried with:
`pod 'WillowTreeReachability’ , ’~> 2.0.0’`
and 
`pod 'WillowTreeReachability'`
with same results.

My config:
Sierra 10.12.5
Xcode 8.3.3

Thank You very much
Kind regards
Fabio

**Comment by eriklamanna** [comment]
@fabioiegri thank you for the issue report. It appears that cocoapods has made some changes to their pod spec format that requires an update. I have a fix and will let you know once it's published to the cocoapods spec repo.

**Comment by fabioiegri** [comment]
Hello Erik.
Thank You very much for fast reply.
Thank You again for the fix. You are the best ones.

Have a nice day
Fabio

**Comment by eriklamanna** [comment]
@fabioiegri this should now be resolved with version 2.0.4

**Comment by fabioiegri** [comment]
Hi guys.
I confirm You that the problem has been solved.

`Installing WillowTreeReachability (2.0.4)`

You are the best ones.

Thank You very much
Kind regards
Fabio
